### PR TITLE
Add eap description and note

### PR DESCRIPTION
**Pull request by mooreds**

### CODE REVIEW

File: site/docs/v1/tech/admin-guide/releases.adoc [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+While the FusionAuth team will make every effort to release an EAP version with stable APIs, they may change. In addition, there is no supported upgrade path from an EAP instance to a released version.
+
+=== Usage Guidelines
+
+An EAP is appropriate for the following purposes:
+
+* testing out features
+* verifying a new FusionAuth version won't break your existing integrations
+* load testing to gain performance insights
+* running end to end tests against to ensure your applications can be safely upgraded
```
Comment (spwitt) [inline site/docs/v1/tech/admin-guide/releases.adoc:63]:
```suggestion
* running end-to-end tests against to ensure your applications can be safely upgraded
```

What are tests running against?

File: site/docs/v1/tech/admin-guide/releases.adoc [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+[WARNING.warning]
+====
+There is no guaranteed upgrade path from an EAP version to a released version.
+
+It should be used only to test features and integrations in order to prepare for an upcoming release.
+
+Seriously.
+====
+
+Some use cases where an EAP instance may be helpful include testing a new install with existing configuration with the EAP version, using the following steps:
```
Comment (spwitt) [inline site/docs/v1/tech/admin-guide/releases.adoc:80]: It says "some use cases," but then this line only mentions one use case.

```suggestion
One use case where an EAP instance may be helpful is testing a new install with existing configuration with the EAP version, using the following steps:
```

**Review by spwitt — APPROVED** [review]

File: site/docs/v1/tech/admin-guide/releases.adoc [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+* migrating a production dataset in a test environment to quantify upgrade durations
+* giving feedback to the FusionAuth team
+
+You **should not use** an EAP to:
+
+* authenticate production users
+
+[WARNING.warning]
+====
+There is no guaranteed upgrade path from an EAP version to a released version.
```
Comment (robotdan) [inline site/docs/v1/tech/admin-guide/releases.adoc:73]: Remove guaranteed- I think we should say - there is not an upgrade path, it is a dead end.

File: site/docs/v1/tech/admin-guide/releases.adoc [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+== Early Access Program
+
+[NOTE.since]
+====
+Available since 1.41.0
+====
+
+FusionAuth has an early access program (EAP).
+
+The EAP allows you, within certain constraints, to test drive the features and functionality of the next release of FusionAuth.
```
Comment (robotdan) [inline site/docs/v1/tech/admin-guide/releases.adoc:35]: Another component of the EAP is that any new features that will require a license at GA - will not require a license during EAP to allow for everyone to test.

**Pull request merged**